snapshot/policy/error_handling_policy.go

```diff
@@ -0,0 +1,37 @@
+package policy
+
+// ErrorHandlingPolicy controls error hadnling behavior when taking snapshots.
+type ErrorHandlingPolicy struct {
+	// IgnoreFileErrors controls whether or not snapshot operation should terminate when a file throws an error on being read
+	IgnoreFileErrors bool `json:"ignoreFileErrs,omitempty"`
```

Comment:
why not spell out "Errors" instead of "Errs" ?

Reply:
✔️